test/e2e/storage/persistent_volumes-local.go

```diff
@@ -356,6 +357,96 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("should support SELinux labeling on local storage volume", func() {
+
+		var testVol *localTestVolume
+
+		BeforeEach(func() {
+			testVol = setupLocalVolumePVCPV(config, node0)
+		})
+
+		AfterEach(func() {
+			cleanupLocalVolume(config, testVol)
+		})
+
+		It("should be able to mount volume in two pods at the same time, write from pod1, and read from pod2 when using the same SELinux label", func() {
+			By("Creating podSc1")
+			podSc1, podSc1Err := createSecPod(config, testVol, false, false, &v1.SELinuxOptions{
+				Level: "s0:c0,c1",
+			})
+			Expect(podSc1Err).NotTo(HaveOccurred())
+
+			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, podSc1))
```

Comment:
no, createSecPod only build spec for a pod which is returned to posSc1 variable and then the actual pod starting point is on line 381-384